packages/volto/news/3839.breaking

```diff
@@ -0,0 +1 @@
+Use default query in SSR rendering and client fetch of the listing block when we don't have a search criteria. Fix https://github.com/plone/volto/issues/3839 @wesleybl
```

Comment:
FYI, by virtue of the name of this file, when this gets included in a release, the process will automatically create an item under the "Breaking" heading with a link to the issue #3839. See for example https://6.docs.plone.org/volto/release-notes/#breaking

```suggestion
Use default query in SSR rendering and client fetch of the listing block when we don't have a search criteria. @wesleybl
```